static/styles/popovers.css

```diff
@@ -317,10 +317,21 @@ ul {
         text-align: center;
         line-height: 30px;
 
-        #edit-button {
+        #edit-button,
+        #modal-edit-user-button {
             font-size: 18px;
             margin-left: 10px;
         }
+
+        #modal-edit-user-button {
+            background: transparent;
+            border: none;
+            color: hsl(200, 100%, 40%);
+
+            &:hover {
+                color: hsl(200, 79%, 66%);
+            }
```

Comment:
What's the thinking around this hover CSS applying only to this component, and not also to `#edit-button`?

Reply:
The `#edit-button` has the hover style implicitly applied from the parent `<a>`, which was why I kept it separate.

Reply:
OK, makes sense. Possibly this kind of detail deserves a comment, but maybe it's a bit in the weeds.